src/controllers/expense.controller.js

```diff
@@ -0,0 +1,68 @@
+const expenseService = require('../services/expense.service');
+const { getById: getUserById } = require('../services/user.service');
+
+const get = async (req, res) => {
+  const expenses = await expenseService
+    .getAll(req.query)
+    .then((exps) => exps.map(expenseService.normalize));
```

Comment:
The use of both `async/await` and `.then()` for handling promises in the `get` function is inconsistent. Consider using `await` for the entire promise chain for better readability and consistency.